ingest/src/main/java/com/microsoft/azure/kusto/ingest/result/TableReportIngestionResult.java

```diff
@@ -17,10 +19,13 @@ public TableReportIngestionResult(List<IngestionStatusInTableDescription> descri
     }
 
     @Override
-    public List<IngestionStatus> getIngestionStatusCollection() {
+    public List<IngestionStatus> getIngestionStatusCollection() throws URISyntaxException {
         List<IngestionStatus> results = new LinkedList<>();
         for (IngestionStatusInTableDescription descriptor : descriptors) {
             TableClient table = descriptor.getTableClient();
+            if (table == null) {
```

Comment:
I'd rather let the IngestionStatusInTableDescription.getTableClient class handle the null case - also it can then save it for later uses